WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -2,7 +2,14 @@ import Yosemite
 
 /// View model for `NewOrder`.
 ///
-final class NewOrderViewModel {
+final class NewOrderViewModel: ObservableObject {
+    private let siteID: Int64
+    private let stores: StoresManager
+
+    /// Closure to be executed when order is created
+    ///
+    private let onCompletion: (Order) -> Void
```

Comment:
How do you plan to use this closure? Sometimes its better to not get tied to a return type early in the process to not add unnecessary constraints in the code 😅

Reply:
That's fair! For now I'm calling it when the order is created successfully (in the success case in `createOrder()`), with the idea that once the order is created remotely we can use it to show the new order right away in the order list. But I can see how it's better to wait and make decisions about that when we're ready to use it.

Reply:
Removed in a00bd05.